### DISCUSSION TITLE
Wikilinks?

### DISCUSSION THREAD

**Discussion by feraxhp** [drl990114/MarkFlowy#692] [Ideas]
Have you planned to add wikilinks support?

**Comment by drl990114** [comment]
Are you referring to adding a GitHub wiki? Or something else?

**Reply by feraxhp** [reply]
Nop, what a mean is that instead of adding a normal (example `![file](file.md)`) link on a md file. use the wikilink option: [[file.md]]

**Reply by drl990114** [reply]
There are no plans for this feature in the short term. We may consider it after some other more important features are completed, but I can't make a commitment to it yet.
